media/src/polls/Polls.svelte

```diff
@@ -70,14 +73,19 @@
     };
 
     const onClickBroadcast = function(e, p) {
+        userResponseList = {};
```

Comment:
I feel like the user response list should be derived out of the pollResponses - one of the changes I'm planning on tomorrow is attaching the session key to the pollResponse object - all the data we need should be in a single array.

Reply:
That's a logical next step, I just put this in place to prevent students from providing multiple responses.